README.md

```diff
@@ -34,14 +34,48 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+    <!-- Answer 1 -->
+
+    Semantic HTML: creates meaning to a web page by making it a more clear, concise, and organized layout.
+
+    <!-- End Answer 1 -->
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+    <!-- Answer 2 -->
+
+    display: block; = takes up entire width of page.
+    dispaly: inline; = takes up only selection.
+
+    <!-- End Answer 2 -->
+
 3. What are the 4 areas of the box model?
 
+    <!-- Answer 3 -->
+
+    1.Content - holds text and images
+    2.Padding - area around content
+    3.border - area around padding
+    4.margin - area around border
+
+    <!-- End Answer 3 -->
+
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
 
+    <!-- Answer 4 -->
+
+    align-items: center =  Verticle axis.
```

Comment:
Be careful with saying verticle or horizontal, because the direction changes depending on if you are using flex-direction: row vs. column!